lxd/networks.go

```diff
@@ -397,7 +398,7 @@ func networkStartup(d *Daemon) error {
 
 		err = n.Start()
 		if err != nil {
-			return err
+			shared.LogError("Failed to bring up network", log.Ctx{"err": err, "name": name})
```

Comment:
It's pretty obvious and the same pattern can be found around container and storage, but sure, I'll add a comment. (Reason is that we don't want LXD to fail to start and enter a respawn loop when a single network fails to start)